db/schemas/schemas.py

```diff
@@ -15,3 +16,11 @@ class Meta:
 
     workflow_status = Enum(Status)
     language = Enum(Language)
+
+
+class JustScoreMetadata(SQLAlchemyAutoSchema):
```

Comment:
🤔 If you like, you can specify in what format to return the DateTimes - see https://marshmallow.readthedocs.io/en/stable/marshmallow.schema.html#marshmallow.schema.Schema.Meta and in particular the `timedateformat` option.